lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -685,6 +685,128 @@ struct ExtractSliceOpConversion
   }
 };
 
+/***
+   # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
```

Comment:
Diff shows this graphical representation in a bad format. You can open the file to see it properly.
I can remove this representation if you think it's too much, but it should at least make review easier.